core/modules/layout/layout.admin.inc

```diff
@@ -29,15 +29,34 @@ function layout_list_page() {
     $row = array();
     // Default layouts have a null path.
     if ($path === '') {
-      $row[] = array('data' => t('Default layouts (used on all paths without a specific layout)'));
+      $row[] = array('data' => t('Default layouts') . ' ' . '<span class="path-parenthetical">' . t('(used on all paths without a specific layout)') . '</span>');
       // Empty columns are intentional so that they may be hidden with
       // priority-low class on the $header array.
       $row[] = '';
       $row[] = '';
     }
     else {
       $path_link = strpos($path, '%') === FALSE ? l($path, $path) : check_plain($path);
-      $row[] = array('data' => $path_link);
+      $cell = array();
+      $page_callback = db_query('
+        SELECT page_callback
+        FROM {menu_router}
+        WHERE path = :path
+        ', array(':path' => $path))
+        ->fetchField();
+      if (!$page_callback) {
+        $cell['class'] = 'missing-path';
+        $path_link .= ' ' . '<span class="path-parenthetical">' .t('(no callback)'). '</span>';
```

Comment:
Have you considered to put the parenthesis outside the translatable string? They are important and always should be there.

Another thought on the css class: How about "callback-missing" or "missing-callback" - IMO parenthetical is a bit misleading here. On the other hand, the parent item class "missing-path" could already be sufficient, to use `.missing-path span` as CSS selector.

@bugfolder what do you think?

Reply:
Thanks for pointing out putting parens outside. I'll make that change.

On the class, my rationale was to use the common class "path-parenthetical" for all parentheticals (which is what allows the parentheticals to be formatted as not-bold); so that includes this one, the (existing) parenthetical on Default Layouts, and possibly any future parenthetical that might be added for reasons we haven't thought of yet. 

And I thought that using the class, rather than the span element, as the CSS selector would then allow the possibility for future changes that might introduce other spans inside of that td (maybe some future tooltip-like thing).